orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -101,18 +101,21 @@ TEST(GradientCheckerTest, SqrtGrad) {
 }
 
 void TestBroadcastableBinaryOpGrad(const std::string& op_type,
-                                   std::function<float(float)>* transformer = nullptr) {
+                                   std::function<float(float)>* transformer = nullptr,
+                                   bool check_not_have_shape_inferencing = false) {
```

Comment:
Can we make this default true? 
So that future boradcasting ops would also handle shape missing case.